src/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -406,7 +406,7 @@ protected function processParams(File $phpcsFile, $stackPtr, $commentStart)
 
             foreach ($typeNames as $typeName) {
                 // Strip nullable operator.
-                if ($typeName[0] === '?') {
+                if (strlen($typeName) > 1 && $typeName[0] === '?') {
```

Comment:
Would it make sense to simplify this to the below ?
```suggestion
                if (isset($typeName[0]) === true && $typeName[0] === '?') {
```

Note: the `=== true` is unnecessary, but a required coding convention in this repo.

Reply:
Both options will work. I don't see either as "more simple." I find `strlen()` to be a little more clear, but am happy to swap this for `isset()` for its better performance.

Reply:
Well, the string length is irrelevant if only looking for the first byte and yes, performance. Either way, your PR.

Reply:
I'm happy to receive feedback. Please don't be discouraged from providing useful insight as you have been. :+1:

Reply:
No worries, I didn't take it like that.